### DISCUSSION TITLE
z teal

### DISCUSSION THREAD

**Discussion by lengyijun** [ajeetdsouza/zoxide#931] [Q&A] [answered]
I often goto `~/.local/share/tealdeer/pages/`
I want `z teal` to jump there, but it doesn't happen.

`zoxide query teal` doesn't show that as well

**Comment by ajeetdsouza** [comment, marked as answer]
The last keyword (in this case, `teal`) should match the final component of your directory (in this case, `pages`).

In this case, you can use queries like:
- `z pages` - matches the final component
- `z te pa` - partially matches the final component
- `z teal /` - implies that `teal` is _not_ the final component
